dev-data/mentorship-data.types.ts

```diff
@@ -0,0 +1,41 @@
+import { COURSE_TITLES } from './courseTitles.data.ts';
+import { ROUTES } from '@/app/const';
+import { Language } from '@/shared/types.ts';
```

Comment:
Please remove the file extension from the path

Reply:
Done